lineman/app/components/flash/flash.scss

```diff
@@ -14,6 +14,25 @@
   padding: 15px;
   width: 100%;
   border-radius: 6px;
+  display: table;
+}
+
+.flash-root__message {
+  display: table-cell;
+  padding-right: 10px;
+}
+
+.flash-root__action {
+  display: table-cell;
```

Comment:
Don't have a strong opinion of table-cell vs flex (due to ignorance more than anything else); I just wanted to make it work the way `media-left` and `media-body` do, which uses table-cell.